packages/server/graphql/mutations/createTaskIntegration.ts

```diff
@@ -112,33 +125,40 @@ export default {
       (userId && teamMembers.find((user) => user.userId === userId)) || {}
 
     const teamDashboardUrl = makeAppURL(appOrigin, `team/${teamId}`)
-    const createdBySomeoneElseComment =
-      userId && viewerId !== userId
-        ? taskIntegrationManager.getCreatedBySomeoneElseComment(
-            viewerName,
-            assigneeName,
-            team.name,
-            teamDashboardUrl
-          )
-        : undefined
-
     const createTaskResponse = await taskIntegrationManager.createTask({
       rawContentStr,
-      integrationRepoId,
-      createdBySomeoneElseComment,
-      context,
-      info
+      integrationRepoId
     })
 
     if (createTaskResponse instanceof Error) {
       return {error: {message: createTaskResponse.message}}
     }
 
+    const {issueId, ...updateTaskInput} = createTaskResponse
+
+    if (
+      userId &&
+      viewerId !== userId &&
+      'addCreatedBySomeoneElseComment' in taskIntegrationManager
```

Comment:
i imagine this'll become non-null in the future, but happy to leave as is for now so we can get this big chunk merged master

Reply:
@mattkrick it is already non-null in the interface, so removing this check in my next PR